src/categories/views.py

```diff
@@ -0,0 +1,27 @@
+from .models import Categories
+from .serializers import CategoriSerializer
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.views import APIView
+from middlewares.authentication import AuthenticationJWT
+
+class CategoryList(APIView):
+    permission_classes = [AllowAny]
+
+    def get(self, request, format=None):
+        snippets = Categories.objects.all()
```

Comment:
why `snippets` variable name? snippet = category ?

Reply:
can I change the variable name from `snippets` ->  `categories` ?

Reply:
Yes. i will change categories.. thank you

Reply:
sửa xong có test lại ko r e?

Reply:
Dạ có test lại bằng postman rồi a